pkg/pb/utils.go

```diff
@@ -66,3 +67,11 @@ func CopyNode(src *Node) (dst *Node) {
 
 	return &node
 }
+
+// PanicOnInvalid panics if NodeType is invalid
+func (nt NodeType) PanicOnInvalid(from string) {
+	// TODO: Remove all references
+	if nt == NodeType_INVALID {
+		panic("INVALID NODE TYPE: " + from)
```

Comment:
Let's maybe initially keep it logging (instead of panic) to find new issues. And before release let's remove these.

Reply:
Sure, let's use that for now.